src/extension/helpers/menuConfigs.ts

```diff
@@ -103,22 +103,23 @@ export async function getTemplatesNames(type: 'piral' | 'pilet', size = 50) {
   const test = /@smapiot\/.*-template-(.*)/;
 
   const templates = await result.data.objects.map((elm: any) => {
-    let { author } = elm.package
-    const { description, name: packageName } = elm.package;
-    
-    if (typeof author === 'string') {
-      author = author;
-    } else if (typeof author === 'object' && author) {
-      author = author.name;
-    } else {
-      author = ''
+    const { author, description, name: packageName } = elm.package;
+
+    function stringifyAuthor(author: string | { name: string }) {
```

Comment:
Don't put this function inside another function. There is no benefit. Make it available via the `utils` module.